src/games/strategy/triplea/settings/battle/options/BattleOptionsSettings.java

```diff
@@ -3,10 +3,11 @@
 import games.strategy.triplea.settings.HasDefaults;
 import games.strategy.triplea.settings.SystemPreferenceKey;
 import games.strategy.triplea.settings.SystemPreferences;
-import games.strategy.triplea.ui.BattleDisplay;
+import games.strategy.triplea.ui.battledisplay.BattleDisplay;
 
 public class BattleOptionsSettings implements HasDefaults {
 
+  static final int DEFAULT_BATTLE_DICE_PER_ROW = 18;
```

Comment:
shouldn't this Constant depend on the users windows size? I think it would be better, if tripleA would try to fill all visible space first before adding scroll bars...